src/temgymbasic/gui.py

```diff
@@ -665,15 +671,15 @@ def sync(self, block: bool = True):
             self.yangleslider.setValue(self.beam.tilt_yx[0])
 
     def build(self) -> Self:
-        num_rays = 64
+        num_rays = 2**20
         beam_tilt_y, beam_tilt_x = self.beam.tilt_yx
         beam_radius = self.beam.radius
 
         vbox = QVBoxLayout()
         # vbox.addStretch()
 
         self.rayslider, _ = labelled_slider(
-            num_rays, 1, 512, name="Number of Rays", insert_into=vbox,
+            num_rays, 1, 2**20, name="Number of Rays", insert_into=vbox,
         )
```

Comment:
Seems like quite a large value for a slider! And will probably slow computation quite a bit...